core/adt/impl/multimap.cpp

```diff
@@ -5,9 +5,10 @@
 
 #include "adt/multimap.hpp"
 
-#include "adt/array.hpp"
+#include "storage/amt/amt.hpp"
 
 namespace fc::adt {
+  using storage::amt::Amt;
 
   Multimap::Multimap(const std::shared_ptr<storage::ipfs::IpfsDatastore> &store)
```

Comment:
Why did you changed the implementation but left `flush` untouched (which still returns cid) at the same time when Array::flush was reimplemented too?